index.scss

```diff
@@ -0,0 +1,72 @@
+$main-background-color: #ECEFF8;
+
+$navbar-color: #4F9CC8;
+
+$card-background-color: #9DBED4;
+
+@mixin centering-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+* {
+  margin: 0;
+  font-family: "Trebuchet MS", sans-serif;
+  box-sizing: border-box;
+}
+
+body{
+  background-color: $main-background-color;
+  nav {
+    background-color: $navbar-color;
+    padding: 10px;
+    position: fixed;
+    width: 100vw;
+    z-index: 10;
+    }
+    h1{
+      padding: 15px 0px 15px 10px;
+      color: white;
+    }
+  .allUserCardContainer {
+    @include centering-container;
+    width: 50vw;
+    margin: 0 auto;
+    .user-header {
+      padding-top: 100px;
+      text-align: left;
+    }
+    .userCard {
+      @include centering-container;
+      background-color: $card-background-color;
+      width: 50vw;
+      border: solid 2px #4F9CC8;
+      padding-bottom: 10px;
+      margin: 20px auto;
+      .avatarImg {
+        width: 150px;
+        height: 150px;
+        object-fit: cover;
+        clip-path: circle(40% at 50% 50%);
+        align-self: center;
+      }
+      .cardUsernameText {
+        align-self: center;
+        padding: 10px;
+      }
+    }
+  }
+  @media screen and (max-width: 500px) {
+    .allUserCardContainer {
+      width: 90vw;
+      .userCard {
+        width: 90vw;
+      }
```

Comment:
it's better to have width 100% and a bit of padding or margin, also it's better if this is set only in 1 place rather than 2 places that have to be kept in sync